@navikt/core/css/darkside/typography.darkside.css

```diff
@@ -204,6 +204,18 @@
 
 .navds-error-message {
   color: var(--ax-text-danger);
+
+  &.navds-error-message--show-icon {
+    display: flex;
+    gap: var(--ax-spacing-1);
+  }
+
+  &.navds-error-message--show-icon svg {
```

Comment:
The svg block could be nested inside `.navds-error-message--show-icon {`

Reply:
I'm now internally confused about when to use nesting and when not 😅 (at least I now will try to always add `&` when nesting, regardless of it being redundant in certain cases).

Reply:
One thing we have talked about previously is avoiding using multiple class names in selectors when not needed. For example, it should be sufficient to use `.navds-error-message--show-icon` instead of `.navds-error-message.navds-error-message--show-icon`. It's not important, it just makes it a bit easier to read.

Reply:
BUT typography.css might be a good place to have an exception to this, since it contains styles for multiple components.

Reply:
Yeah, that makes sense, at least that's the feature we get by following BEM (when not thinking about nesting), that each class is unique, and since each component using a `E` or `M` would _always_ (when humans remember) have the `B` on it too, meaning they are "additive" too. (I don't super enjoy that there's a fair amount of human error potential with the BEM convention 😓, but seems like there's no clearly better alternative out there?).